src/com/facebook/buck/android/AndroidPrebuiltAar.java

```diff
@@ -119,6 +121,12 @@ public Path getBinaryJar() {
     return prebuiltJar.getPathToOutput();
   }
 
+  @Nullable
+  @Override
+  public Path getPathToOutput() {
+    return unzipAar.getPathToOutput();
```

Comment:
It feels a bit round-about to use the `UnzipAar` rule to thread through the original AAR (but it's likely I'm missing something).  Can we just have the constructor of this rule take the orignal AAR `SourcePath` and list that as the output without touching `UnzipAar` (and also propagate the appropriate runtime dep on any rule that generates the original AAR)?